src/npcmove.cpp

```diff
@@ -686,6 +713,22 @@ npc_action npc::address_needs(int danger)
         return npc_eat;
     }
 
+    // TODO: More risky attempts at sleep when exhausted
+    if( danger == 0 && fatigue > 191 ) {
+        if( !is_following() ) {
+            fatigue = 0; // TODO: Make tired NPCs handle sleep offscreen
+            return npc_undecided;
+        }
+
+        if( has_effect( "allow_sleep" ) || fatigue > 1000 ) {
+            return npc_sleep;
+        } else if( g->u.sees( *this ) && !has_effect( "npc_said" ) &&
+                   one_in( 10000 / ( fatigue + 1 ) ) ) {
+            say( _("<yawn>") );
```

Comment:
I think you don't need / shouldn't translate those tags, it's not done with other tags (e.g. `say("<lets_talk>")`).